src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,11 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        for (int i = 0; i < machines.length; i++) {
+            machines[i].doWork();
+            machines[i].stopWork();
+        }
+    }
+    static Machine[] machines = new Machine[]{new Truck(), new Bulldozer(), new Excavator()};
```

Comment:
add the access modifier and remind that there should be class fields first and then methods